docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -26,7 +26,12 @@ weight: 80
 
 # Configure data links
 
-You can use data link variables or data links to create links between panels.
+Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization showed four servers, you could add a data link to one or two of them.
+
+The link itself is accessible in different ways depending on the visualization. For the Graph you need to click on a data point or line, for a panel like
```

Comment:
We probably want to refer to time series vs graph (as we are deprecating graph panel in v11)

```suggestion
The link itself is accessible in different ways depending on the visualization. For the Time series you need to click on a data point or line, for a panel like
```

Reply:
I'll make sure to update this in the PR where i'm making copy edits!